crmsh/sbd.py

```diff
@@ -150,8 +151,12 @@ def _initialize_sbd(self):
         """
         if self.diskless_sbd:
             return
+        opt = ""
+        if self._context.profiles_dict:
+            self._sbd_msgwait = self._context.profiles_dict["sbd_msgwait"]
+            opt = "-4 {}".format(self._sbd_msgwait)
```

Comment:
@gao-yan Do we need use `-1` option to setup watchdog timeout when creating sbd? Or, just write the SBD_WATCHDOG_TIMEOUT in /etc/sysconfig/sbd?
What's different here?

Reply:
This need something like
`sbd -d $SBD_DEVICE -4 $sbd_msgwait -1 $sbd_watchdog_timeout create`

Keep in mind, watchdog_timeout on disk overrides `/etc/sysconfig/sbd SBD_WATCHDOG_TIMEOUT`

Reply:
Changed, now the dump output:
```
# sbd -d /dev/sdb1 dump
==Dumping header on disk /dev/sdb1
Header version     : 2.1
UUID               : c1489a11-02f6-4475-ac98-daa5e119d44e
Number of slots    : 255
Sector size        : 512
Timeout (watchdog) : 60
Timeout (allocate) : 2
Timeout (loop)     : 1
Timeout (msgwait)  : 120
==Header on disk /dev/sdb1 is dumped
```